### ISSUE TITLE
Importer and others questions

### ISSUE THREAD

**Issue by ghost** [zachmu/tiled-xna#4]
Hi,

I'm having an issue when I try to build my project. Visual says that it can't detect automatically the importer for my tmx file.

I simply paste the file Tiled.cs in my project.

My map is that :

<?xml version="1.0" encoding="UTF-8"?>
<map version="1.0" orientation="orthogonal" width="100" height="100" tilewidth="32" tileheight="32">
 <properties>
  <property name="mapmeta" value="mapvalue"/>
 </properties>
 <tileset firstgid="1" name="platforms" tilewidth="32" tileheight="32">
  <image source="platforms.png" width="1536" height="64"/>
 </tileset>
 <layer name="Background" width="100" height="100">
  <data encoding="base64" compression="gzip">
   H4sIAAAAAAAAC+3DAQkAAAwDoPH+oV9jMAUTAAAAAAAAYMmVBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIBWD9npg0lAnAAA
  </data>
 </layer>
 <objectgroup name="Collide" width="100" height="100">
  <object name="ground" x="32" y="576" width="1088" height="32"/>
 </objectgroup>
 <objectgroup name="Position" width="100" height="100">
  <object name="start" x="160" y="480" width="32" height="32"/>
 </objectgroup>
</map>

I also have somes questions. In your TiledExample, you are using Square.Tiled but I can't find what is it so I can't use it in my project and I wonder if that the reason of my first issue.

**Comment by ghost** [comment]
My Bad... In properties, Visual tried to compile the tmx file x). So it works !

**Comment by zachmu** [comment]
Yeah, you have to make sure XNA doesn't try to compile your maps.
